opentelemetry-exporter-cloud-monitoring/tests/test_cloud_monitoring_exporter_integration.py

```diff
@@ -0,0 +1,83 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import socket
+import subprocess
+import unittest
+
+import grpc
+from google.cloud.monitoring_v3 import MetricServiceClient
+from google.cloud.monitoring_v3.gapic.transports import (
+    metric_service_grpc_transport,
+)
+from opentelemetry.exporter.cloud_monitoring import (
+    WRITE_INTERVAL,
+    CloudMonitoringMetricsExporter,
+)
+from opentelemetry.sdk import metrics
+from opentelemetry.sdk.metrics.export import MetricRecord, MetricsExportResult
+from opentelemetry.sdk.metrics.export.aggregate import SumAggregator
+
+
+# TODO: #46
+# Refactor duplicated code found in test_integration_cloud_trace_exporter.py
+class BaseExporterIntegrationTest(unittest.TestCase):
+    def setUp(self):
+        self.project_id = "TEST-PROJECT"
+
+        # Find a free port to spin up our server at.
+        sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+        sock.bind(("localhost", 0))
+        self.address = "localhost:" + str(sock.getsockname()[1])
+        sock.close()
+
+        # Start the mock server.
+        args = ["mock_server", "-address", self.address]
+        self.mock_server_process = subprocess.Popen(
```

Comment:
Why is this `stderr=subprocess.PIPE` and not `stdout=subprocess.PIPE`? It seems strange that your server is outputting the address to error?

Reply:
Note sure, to be honest. I was following the precedence from [this](https://github.com/GoogleCloudPlatform/opentelemetry-operations-python/pull/37).

Reply:
Hm, I guess its not a huge deal